content/en/blog/2021-05-31-june-update.md

```diff
@@ -0,0 +1,208 @@
+---
+author: dholbach
+date: 2021-05-31 08:30:00+00:00
+title: June 2021 update
+description: Flux 0.14 and Flagger 1.11 with lots of new features, KubeCon talks galore, Flux at CNCF Store, lots of Flux adoption. We are reviewing the Flux governance.
```

Comment:
I'm thinking the short phrase "we are reviewing Flux governance" could be interpreted to mean something different – like perhaps we are rethinking governance, or it wasn't fully established yet, etc. While a 6 month review feels more planned and standard, as is the case. WDYT?

```suggestion
description: Flux 0.14 and Flagger 1.11 with lots of new features, KubeCon talks galore, Flux at CNCF Store, lots of Flux adoption. Flux governance 6 month review.
```